semgrep/semgrep/rule.py

```diff
@@ -104,6 +104,10 @@ def id(self) -> str:
     def message(self) -> str:
         return str(self._raw["message"])
 
+    @property
+    def metadata(self) -> Dict[str, Any]:  # type: ignore
```

Comment:
I'm not too happy about this ignore but apparently it's not allowed to use `Any` under a decorator? 🤨